exercises/testexam/ex_rnw/ex_poly.Rnw

```diff
@@ -0,0 +1,7 @@
+State the hypothesis space for a univariate polynomial regression model with
```

Comment:
Die Aufgabe verstehe ich nicht ganz, was machen diese Faktoren? Gehen doch im Endeffekt eh in den thetas auf? Formulierung müssten wir dann auch noch überarbeiten.